src/gpu/gpu_matmul_list.cpp

```diff
@@ -39,6 +40,7 @@ namespace {
 constexpr impl_list_item_t impl_list[] = REG_MATMUL_P({
         GPU_INSTANCE_INTEL(intel::ocl::gemm_matmul_t)
         GPU_INSTANCE_INTEL_REF(intel::ocl::ref_matmul_t)
+        GPU_INSTANCE_NVIDIA(nvidia::cudnn_matmul_lt_t)
```

Comment:
The cublasLt implementation will only be selected for the specific IMMA case or when the bias/relu can be run as epilogues. Otherwise it would default to the next best implementation.